### PR TITLE
Update to ungoogled-chromium 128.0.6613.84

### PR DESCRIPTION
**Pull request by Cubik65536**
Notes:

- A submodule bump is made.
- A patch refresh is made.
- Rust is updated according to Chromium requirement.
- The patch for addressing Ninja 1.12 Dependency issue is now removed because it is no longer needed.
- The GitHub Release Note generation script is modified so if there is an `announcements.md` file, an announcement with the file content will be added to the release.
- An announcement related to sponsorship for notarizing builds is added. (https://github.com/ungoogled-software/ungoogled-chromium-macos/issues/179#issuecomment-2299288584)

---

Builds and runs fine locally.

CleanShot 2024-08-23 at 21 53 49

---

Signed-off-by: Qian Qian "Cubik"‎ <cubik65536@cubik65536.top>

### CODE REVIEW

**Comment by PF4Public** [comment]
@networkException Is it how sponsorship announcement supposed to look like? On one hand it is not in readme, so not immediately visible, on the other hand I have no idea how one could make it otherwise.

File: announcements.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Many people have asked for notarizing (signing) the Ungoogled-Chromium macOS build ([ungoogled-chromium#859](https://github.com/ungoogled-software/ungoogled-chromium/issues/859), [#63](https://github.com/ungoogled-software/ungoogled-chromium-macos/issues/63), [#179](https://github.com/ungoogled-software/ungoogled-chromium-macos/issues/179)).
+
+As most of the Ungoogled-Chromium maintainers are maintaining the software on a voluntary basis, we unfortunately do not have the funds for the Apple Developer Program, which is required for notarization. However, we are seeking for sponsors to help us with the costs.
```
Comment (networkException) [inline announcements.md:3]: I'd say all maintainers do :)

Comment (Cubik65536) [reply announcements.md]: Oops, changed it.

**Comment by networkException** [comment]
> @networkException Is it how sponsorship announcement supposed to look like? On one hand it is not in readme, so not immediately visible, on the other hand I have no idea how one could make it otherwise.

Yea this is fine, it can also be in the readme I guess.

In general I think the wording could be more "I, the maintainer, am looking to get update work and the developer account sponsored" instead of "we are looking for sponsorships through this individual's account". I don't think its a shame to bring up donations as a topic for free maintenance work, especially if that has costs attached.

I'd also like to avoid this sounding too much like a project wide thing as there's no legal entity or the likes to accept donations.

**Comment by Cubik65536** [comment]
> > @networkException Is it how sponsorship announcement supposed to look like? On one hand it is not in readme, so not immediately visible, on the other hand I have no idea how one could make it otherwise.
> 
> Yea this is fine, it can also be in the readme I guess.
> 
> In general I think the wording could be more "I, the maintainer, am looking to get update work and the developer account sponsored" instead of "we are looking for sponsorships through this individual's account". I don't think its a shame to bring up donations as a topic for free maintenance work, especially if that has costs attached.
> 
> I'd also like to avoid this sounding too much like a project wide thing as there's no legal entity or the likes to accept donations.

I can change it and add this to README :)

**Comment by Cubik65536** [comment]
I changed the second and third paragraphs to:

> As all Ungoogled-Chromium maintainers are maintaining the software on a voluntary basis, we unfortunately do not have the funds for the Apple Developer Program membership, which is required for notarization.
> 
> As of now, I, [@Cubik65536](https://github.com/Cubik65536), the maintainer of Ungoogled-Chromium macOS, am seeking sponsors to help me cover the cost of the Apple Developer Program fee. If you're interested into getting the "official" Ungoogled-Chromium macOS build notarized, please consider sponsoring me through [GitHub Sponsors](https://github.com/sponsors/Cubik65536) or [Buy me a Coffee](https://buymeacoffee.com/cubik65536).

Would this be better? /cc @networkException

**Comment by networkException** [comment]
Yep thats good

**Review by networkException — DISMISSED** [review]

**Comment by Cubik65536** [comment]
Sorry, just a small fix.

**Review by networkException — APPROVED** [review]

**Pull request merged**